scripts/mutect2_wdl/mutect2_pon.wdl

```diff
@@ -128,6 +131,7 @@ task CreatePanel {
       File gnomad
       File gnomad_idx
       String? create_pon_extra_args
+      String? gcs_project_for_requester_pays
```

Comment:
I don't think this is connected to anything.  Should this argument be used on line 152 and 154?

Reply:
Indeed.  After I asked the gnomAD team and they said it's not, nor is it ever expected to be requester pays, I forgot to take this out.